public/editor/scripts/editor/js/fc/bramble-ui-bridge.js

```diff
@@ -222,6 +222,22 @@ define(function(require) {
       }
     }
 
+    $(".fullscreen-preview-toggle").click(function() {
+      toggleFullscreenPreview();
+    });
+
+    function toggleFullscreenPreview(){
+      if($("body").hasClass("fullscreen-preview")){
+        analytics.event("Fullscreen Off");
```

Comment:
The rest of the analytics events have no spaces in the name--not sure if that matters or not, but let's be consistent: `FullscreenOn` and `FullscreenOff`.